boards/components/src/nmea.rs

```diff
@@ -0,0 +1,121 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2022.
+
+//! Component for any NMEA data.
```

Comment:
This is actually a component for a specific I2C GNSS device, not a generic NMEA device. Please rename the component and fix this comment.